src/Core/Migration/V6_3/Migration1590408550AclResources.php

```diff
@@ -70,7 +70,7 @@ private function getRoles(Connection $connection): array
         return $grouped;
     }
 
-    private function tableExists(Connection $connection, string $table): bool
+    protected function tableExists(Connection $connection, string $table): bool
```

Comment:
why not removing, now that the method is in the parent class?

Reply:
All good 😄